src/sentry/utils/metrics.py

```diff
@@ -130,3 +131,13 @@ def timer(key, instance=None, tags=None):
         tags['result'] = 'success'
     finally:
         timing(key, time() - start, instance, tags)
+
+
+def wraps(key, instance=None, tags=None):
```

Comment:
This is one of the few cases I feel like using `*args, **kwargs` would actually be justified (but I don't really have a strong preference either way, this obviously works fine too.)

Reply:
I think it'd be good to write a trivial test for this just to make sure the decorator works correctly (not specifically testing the timing, but making sure it returns the value from the wrapped function to avoid regressions.)

Reply:
Did it.